freqtrade/data/converter.py

```diff
@@ -33,7 +39,41 @@ def parse_ticker_dataframe(ticker: list) -> DataFrame:
     })
     frame.drop(frame.tail(1).index, inplace=True)     # eliminate partial candle
     logger.debug('Dropping last candle')
-    return frame
+
+    if fill_missing:
```

Comment:
a) fill_missing is not True everywhere, there is 1 (or 2) tests where it's false. I can circumvent these tests easily, but didn't see the need.
It may not be necessary, but my thinking was to eventually add a config-flag to allow disabling (although the speed-penalty was small enough so I dropped that idea for now).
Additionally - `from freqtrade.data import converter` can be a good analysis method - and skipping there to analyze what is missing in your backtest data CAN be useful - and that's why I left it in.

b)`len(ticker) % interval == 0` does not return correct results.

`len(ticker)` is supposed to have 500 candles (actually 499, because dropped last candle) - so it will show always "wrong" for 5m ticker, even though not 1 single candle is missing.
it could have 500 candles `len(ticker) == 500`, and 20 in the middle are missing, or 1 is missing, ... - and that's not that easy to detect, and is not obvious (and probably not identical between exchanges) if it returns 500 (skipping 20 missing) - or if it would return 480 in that case.

It is possible by comparing `first -last / len(ticker)` (i think) - but i'm not sure this "complication" makes sense.